vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -52,226 +52,236 @@ import (
 func (client *Client) Close() error {
-	return client.conn.Close()
+    return client.conn.Close()
 }
 
 // Path templates.
 
 // Shelf represents a shelf resource.
 type Shelf struct {
-	Shelf string
+    Shelf string
 }
 
 // NewShelfFromPath creates a Shelf from a shelf path in the form of "shelves/{shelf}".
 func NewShelfFromPath(path string) (Shelf, error) {
-	binding, err := shelfPathTemplate.Match(path)
-	if err != nil {
-		return Shelf{}, err
-	}
-	return Shelf{
-		Shelf: binding["shelf"],
-	}, nil
+    binding, err := shelfPathTemplate.Match(path)
+    if err != nil {
+        return Shelf{}, err
+    }
+    return Shelf{
+        Shelf: binding["shelf"],
+    }, nil
 }
 
 // Path returns the path for the shelf resource.
 func (shelf Shelf) Path() (string, error) {
-	return shelfPathTemplate.Instantiate(map[string]string{
-		"shelf": shelf.Shelf,
-	})
+    return shelfPathTemplate.Instantiate(map[string]string{
+        "shelf": shelf.Shelf,
+    })
 }
 
 // Book represents a book resource.
 type Book struct {
-	Shelf string
-	Book string
+    Shelf string
+    Book string
 }
 
 // NewBookFromPath creates a Book from a book path in the form of "shelves/{shelf}/books/{book}".
 func NewBookFromPath(path string) (Book, error) {
-	binding, err := bookPathTemplate.Match(path)
-	if err != nil {
-		return Book{}, err
-	}
-	return Book{
-		Shelf: binding["shelf"],
-		Book: binding["book"],
-	}, nil
+    binding, err := bookPathTemplate.Match(path)
+    if err != nil {
+        return Book{}, err
+    }
+    return Book{
+        Shelf: binding["shelf"],
+        Book: binding["book"],
+    }, nil
 }
 
 // Path returns the path for the book resource.
 func (book Book) Path() (string, error) {
-	return bookPathTemplate.Instantiate(map[string]string{
-		"shelf": book.Shelf,
-		"book": book.Book,
-	})
+    return bookPathTemplate.Instantiate(map[string]string{
+        "shelf": book.Shelf,
+        "book": book.Book,
+    })
 }
 
 // AUTO-GENERATED DOCUMENTATION AND METHOD -- see instructions at the top of the file for editing.
 
 // CreateShelf creates a shelf, and returns the new Shelf.
 func (client *Client) CreateShelf(ctx context.Context, req *google_example_library_v1.CreateShelfRequest, opts ...gax.CallOption) (*google_example_library_v1.Shelf, error) {
-	callOpts := append(nonIdempotentRetryCodes(), defaultRetryParams(), client.CallOptions["CreateShelf"]...)
-	callOpts = append(callOpts, opts...)
-	var resp *google_example_library_v1.Shelf
-	err := gax.Invoke(ctx, func (c context.Context) error {
-		var err error
-		resp, err = client.client.CreateShelf(c, req)
-		return err
-	}, callOpts...)
-	if err != nil {
-		return nil, err
-	}
-	return resp, nil
+    var callOpts []gax.CallOptions
+    callOpts = append(callOpts, nonIdempotentRetryCodes())
+    callOpts = append(callOpts, defaultRetryParams())
+    callOpts = append(callOpts, client.callOptions["CreateShelf"])
+    callOpts = append(callOpts, opts...)
```

Comment:
I just realized we can use gax.callOptions if it were exported. I'm not sure if it's worth changing at this point.